src/Symfony/Component/Console/Style/SymfonyStyle.php

```diff
@@ -110,7 +110,7 @@ public function title($message)
         $this->autoPrependBlock();
         $this->writeln(array(
             sprintf('<comment>%s</>', $message),
-            sprintf('<comment>%s</>', str_repeat('=', strlen($message))),
+            sprintf('<comment>%s</>', str_repeat('=', Helper::strlenWithoutDecoration($message))),
```

Comment:
You should pass the return value of `$this->getFormatter()` as the first argument to `Helper::strlenWithoutDecoration()` (same below).